main.go

```diff
@@ -94,6 +95,7 @@ type globalMetricConfig struct {
 
 // Nas CSI Plugin
 func main() {
+	flag.Var(features.FunctionalMutableFeatureGate, "featuregate", "Enable or disable specific feature gates")
```

Comment:
https://github.com/kubernetes-csi/external-provisioner/blob/33c5c1f8f2f78e7756969cf37c2417c339c4c216/cmd/csi-provisioner/csi-provisioner.go#L131-L132

Be consistent with other CSI sidecars?

Reply:
These are just the different methods of parsing input parameters. Will it cause consistency issues?

Reply:
my largest concern is the flag name, `feature-gates` vs `featuregate`

Reply:
I misunderstand, you're right. `feature-gates` is reasonable.